app/src/ui/merge-branch/merge.tsx

```diff
@@ -258,16 +257,12 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
         id="merge"
         onDismissed={this.props.onDismissed}
         onSubmit={this.merge}
+        title={
+          <div className="merge-dialog-header">
```

Comment:
In a similar vein to the rebase dialog I don't think we need this wrapping element any more. We should be able to change this to `<>Merge into <strong>${currentBranchName}</strong></>` and update the CSS just like we did in the rebase dialog by moving the `.merge-dialog-header` selector into the the `.dialog-header` selector and changing it to `h1`. Something like this

https://github.com/desktop/desktop/blob/6206729db9212815a14fe761637be28a036c01d2/app/styles/ui/dialogs/_rebase.scss#L13-L19